### PR TITLE
Prep open source

### PR DESCRIPTION
**Pull request by JeroenKnoops**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Contributing to NPM dependencies extractor 
+
+We'd love for you to contribute to our source code and to make the Forest even better than it is today! Here are the guidelines we'd like you to follow:
```
Comment (sandacontiume) [inline CONTRIBUTING.md]: Replace Forest -> npm-dependencies-extractor

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+ - [Question or Problem?](#question)
+ - [Issues and Bugs](#issue)
+ - [Feature Requests](#feature)
+ - [Submission Guidelines](#submit)
+ - [Further Info](#info)
+
+## <a name="question"></a> Got a Question or Problem?
+
+If you have questions about how to use the Forest, please direct these to the [Slack group / philips-software][slack].
```
Comment (sandacontiume) [inline CONTRIBUTING.md]: Replace Forest -> npm-dependencies-extractor

**Review by sandacontiume — CHANGES_REQUESTED** [review]
Please solve the 2 renaming remarks, then I can merge

File: CONTRIBUTING.md (hunk 1 above)
Comment (JeroenKnoops) [reply CONTRIBUTING.md]: Done

File: CONTRIBUTING.md (hunk 2 above)
Comment (JeroenKnoops) [reply CONTRIBUTING.md]: Done

**Review by sandacontiume — APPROVED** [review]
agree

**Pull request merged**